src/book.py

```diff
@@ -314,9 +314,9 @@ def _read_coinbase(self, file_path: Path) -> None:
                 utc_time = utc_time.replace(tzinfo=datetime.timezone.utc)
                 operation = operation_mapping.get(operation, operation)
                 change = misc.force_decimal(_change)
-                #  Current price from exchange.
-                eur_spot = misc.force_decimal(_eur_spot)
-                #  Cost without fees.
+                # Rounded price from CSV
+                # eur_spot = misc.force_decimal(_eur_spot)
+                # Cost without fees
                 eur_subtotal = misc.xdecimal(_eur_subtotal)
```

Comment:
Why is `xdecimal` (may be None) and `eur_spot` is always a decimal.

When `eur_splot` is always given, `eur_subtotal` should also be always given?

Reply:
Yes, it is always given for my CSV logs. I also wondered about the same thing, but did not want to change `xdecimal` to `force_decimal` (it may have been written like that for a reason). We could change it, but I'm not sure if this breaks it for other CSV logs.

Reply:
They can open a bug request in that case ;) Change it to `force_decimal` for now.